MAINTAINERS_RULES.md

```diff
@@ -0,0 +1,39 @@
+# Maintainers Rules
+
+This document lays out some basic rules and guidelines all maintainers are expected to follow.
+Changes to the [Acceptance Criteria](#hard-acceptance-criteria-for-merging-a-pr) for merging PRs require a ceiling(two-thirds) supermajority from the maintainers.
+Changes to the [Repo Guidelines](#repo-guidelines) require a simple majority.
+
+## Hard Acceptance Criteria for merging a PR:
+
+- 2 LGTMs are required when merging a PR
+- If there is obviously still discussion going on in the PR, even with 2 LGTMs, let the discussion resolve before merging. If you’re not sure, reach out to the maintainers involved in the discussion.
+- All checks must be green 
+    - There are limited mitigating circumstances for this, like if the docs builds are just broken and that’s the only test failing.
```

Comment:
instead of listing exceptions to this rule, it may be more useful to state when the `[skip ci]` flag should be used. We don't really have another way to turn off the checks. Besides the build, i think the only other check we have is that it's marked as approved, which we want anyway.